docs/packages/TypeScript_ESLint.mdx

```diff
@@ -50,31 +53,52 @@ export default tseslint.config(
 );
 ```
 
+This config file exports a flat config that enables both the [core eslint recommended config](https://www.npmjs.com/package/@eslint/js) and [our recommended config](../users/Shared_Configurations.mdx#recommended).
+
+For more information on the `tseslint.config` function [see below](#config).
+
+### Advanced usage
+
 You can also declare our plugin and parser in your config via this package. For example this config would enable our plugin, our parser, and type-aware linting with a few of our popular type-aware rules:
 
 ```js title="eslint.config.js"
 // @ts-check
 
+import eslint from '@eslint/js';
+import importPlugin from 'eslint-plugin-import';
```

Comment:
**sugestion:** I think it's a great idea to include and example config object using the `import` plugin but you're missing one in the config below below.

Reply:
oops this was supposed to be
```suggestion
import jestPlugin from 'eslint-plugin-jest';
```

I actually tossed up between `jest` and `import`.
`import` is a lot more used (~21m vs ~10m) so it's more likely it would "resonate" with people.
However it's rare that you'd configure `import` rules in a separate config object. I.e. there's no reason you'd turn on `import/consistent-type-specifier-style` on just one type of file.

I really wanted a second config object here so that I could very clearly illustrate that the function can take multiple objects.

This is ultimately went with `jest` because it gave me a excuse to add a non-contrived, real world example of a second config object because the common usecase is to just turn its rules on on test files.